modules/sriov-operator-metrics.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="sriov-operator-metrics_{context}"]
+= Enabling the SR-IOV Network Operator metrics
+
+You can enable the SR-IOV Network Operator metrics to monitor and analyze the Operator's performance. 
+
+The SR-IOV Network Operator metrics are disabled by default. To enable them, you must set the `spec.featureGates.metricsExporter` field to `true`. 
+
+Following are the SR-IOV Network Operator metrics:
+
+* `sriov_vf_rx_broadcast`
+* `sriov_vf_rx_bytes`
+* `sriov_vf_rx_dropped`
+* `sriov_vf_rx_multicast`
+* `sriov_vf_rx_packets`
+* `sriov_vf_tx_bytes`
+* `sriov_vf_tx_dropped`
+* `sriov_vf_tx_packets`
+* `sriov_kubepoddevice`
+
+.Example PromQL queries to examine the SR-IOV Network Operator metrics:
+[source,bash]
+----
+sriov_vf_tx_packets * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice
+sriov_vf_rx_packets * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice
+sriov_vf_tx_bytes * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice
+sriov_vf_rx_bytes * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice
+sriov_vf_tx_dropped * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice
+sriov_vf_rx_dropped * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice
+sriov_vf_rx_broadcast * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice
+sriov_vf_rx_multicast * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice
+----
+
+.Prerequisites
+
+* You have installed the {oc-first}.
+* You have logged in as a user with `cluster-admin` privileges.
+* You have installed the SR-IOV Network Operator.
+
+.Procedure
+
+* Set the `spec.featureGates.metricsExporter` field to `true` by running the following command:
++
+[source,terminal]
+----
+$ oc patch -n openshift-sriov-network-operator sriovoperatorconfig/default \
+    --type='merge' -p='{"spec": {"featureGates": {"metricsExporter": true}}}'
+----
```

Comment:
Please, add the namespace labeling step:
```
oc label ns/openshift-sriov-network-operator openshift.io/cluster-monitoring=true
```

This is needed for the feature to be activated

Reply:
Updated as per the suggestion.